lnwallet/channel.go

```diff
@@ -9087,3 +9087,19 @@ func (lc *LightningChannel) MultiSigKeys() (keychain.KeyDescriptor,
 	return lc.channelState.LocalChanCfg.MultiSigKey,
 		lc.channelState.RemoteChanCfg.MultiSigKey
 }
+
+// LocalCommitmentBlob returns the custom blob of the local commitment.
+func (lc *LightningChannel) LocalCommitmentBlob() fn.Option[tlv.Blob] {
```

Comment:
In these situations I always ask myself, why there isn't a remote corollary to this API?

Reply:
I think in this case, we use it within the router, and we're usually interested in our local balance.